code/game/gamemodes/objective_items.dm

```diff
@@ -65,12 +65,6 @@
 	difficulty = 3
 	excludefromjob = list("Head of Security", "Warden")
 
-/datum/objective_item/steal/reactive
```

Comment:
I personally think its stupid... I mean I dont know any other objectives that take time to research up to.